assets/embauche.html

```diff
@@ -50,7 +50,16 @@
 					<option>mois</option>
 					<option disabled>jour</option>
 					<option disabled>heure</option>
-				</select>.
+				</select>
+
+				sur la commune de
+				<input name="code_postal_entreprise" id="code_postal_entreprise" class="medium-width centered"
+				type="text" placeholder="code postal" pattern="\d{5}" title="Entrez le code postal de l'établissement où le salarié sera employé">
```

Comment:
type="number" is not appropriate for a postal code (because of the up and down arrows it introduces).
I can't find references about setting custom `type` attributes... 
inputmode is cool for mobiles :+1: 

Placeholders can also be a "brief description of the expected format". I believe it's clearer to see "code postal" than for instance "29200". Indeed, "commune de" should probably be the label of this field.